packages/core/src/messagingProtocols/proteus/CryptMessages/CryptMessages.ts

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
CryptMessage, this name doesn't make sense. We need to find something better there